src/uielement_button.js

```diff
@@ -0,0 +1,73 @@
+/**
+A UIElement that shows a button.
+The provided callback function gets called if a onclick event occurs on this button.
+
+@class UIElementButton
+@augments UIElement
+
+@param {string} [className] CSS class
+@param {string} caption Caption of the button
+@param {method} actionFunction Callback function for onclick event
+*/
+function UIElementButton(className, caption, actionFunction) {
```

Comment:
@actionFunction: you are right - it is a callback (as stated in line 3).
Better name would be actionCallback.

Reply:
Fixed.